generator/consts.py

```diff
@@ -34,13 +34,15 @@
 iso_639_languages = {
     "BG": "Bulgarian",
     "BS": "Bosnian",
-    "CS": "Czech",
+    "CA": "Catalan",
+    "CZ": "Czech",
     "DA": "Danish",
     "DE": "German",
     "EN": "English",
     "ES": "Spanish",
     "FI": "Finnish",
     "FR": "French",
+    "GR": "Greek",
```

Comment:
This should be `EL`. As mentioned in another comment: if the csv is wrong we should correct it there.